integration_tests/testsuite/test_util.py

```diff
@@ -140,3 +141,7 @@ def _project_id():
 
 def get_default_url():
     return 'https://{0}.appspot.com'.format(_project_id())
+
+
+def generate_version():
```

Comment:
Maybe put an identifying string in the version name, something like:

integrationtest-$DATETIME